lib/api/2.0/skus.js

```diff
@@ -0,0 +1,164 @@
+    return   skuPack.getSkusById(req.swagger.params.identifier.value);
+});
+
+/**
+ * @api {get} /api/2.0/skus/:identifier/nodes GET /:id/nodes
+ * @apiVersion 2.0.0
+ * @apiDescription get nodes for specific sku
+ * @apiName sku-get-nodes
+ * @apiGroup skus
+ * @apiParam {String} identifier of sku, must cast to ObjectId
+ * @apiError NotFound There is no sku with the <code>identifier</code>
+ * @apiErrorExample Error-Response:
+ *     HTTP/1.1 404 Not Found
+ *     {
+ *       "error": "Not Found"
+ *     }
+ */
+var skusIdGetNodes = controller(function(req) {
+    return   skuPack.getNodesSkusById(req.swagger.params.identifier.values);
+});
+
+/**
+ * @api {post} /api/2.0/skus POST /
+ * @apiVersion 2.0.0
+ * @apiDescription create a sku
+ * @apiName sku-post
+ * @apiGroup skus
+ * @apiError E_VALIDATION attributes are invalid.
+ */
+var skusPost = controller({success: 200}, function(req) {
+    return   skuPack.postSku(req.body);
+});
+
+/**
+ * @api {post} /api/2.0/skupacks POST /
+ * @apiVersion 2.0.0
+ * @apiDescription create a sku following the rules in the pack
+ * @apiName skupacks-post
+ * @apiGroup skupacks
+ * @apiError E_VALIDATION attributes are invalid.
+ */
+var skuPackPost = controller(function(req, res) {
+    return   skuPack.skuPackHandler(req,res);
+});
+
+/**
+ * @api {patch} /api/2.0/skus/:identifier PATCH /:id
+ * @apiVersion 2.0.0
+ * @apiDescription patch specific sku
+ * @apiName sku-patch
+ * @apiGroup skus
+ * @apiParam {String} identifier of sku, must cast to ObjectId
+ * @apiError NotFound There is no sku with the <code>identifier</code>
+ * @apiErrorExample Error-Response:
+ *     HTTP/1.1 404 Not Found
+ *     {
+ *       "error": "Not Found"
+ *     }
+ */
+var skusPatch = controller( function (req) {
+    return   skuPack.patchSku(req.swagger.params.identifier.value, req.body);
+});
+
+/**
+ * @api {put} /api/2.0/skus/:identifier/pack PUT /:id/pack
+ * @apiVersion 2.0.0
+ * @apiDescription post a sku pack to specific sku
+ * @apiName sku-pack-id-put
+ * @apiGroup skus
+ * @apiParam {String} identifier of sku, must cast to ObjectId
+ * @apiError NotFound There is no sku with the <code>identifier</code>
+ * @apiErrorExample Error-Response:
+ *     HTTP/1.1 500 Internal Server Error
+ *     {
+ *       "error": "error message"
+ *     }
+ */
+var skusIdPutPack = controller( {success: 201}, function (req,res) {
+    return   skuPack.putPackBySkuId(req,res);
+});
+
+/**
+ * @api {delete} /api/2.0/skus/:identifier/pack DELETE /:id
+ * @apiVersion 2.0.0
+ * @apiDescription Delete a pack for a specific sku.
+ * @apiName sku-delete-pack
+ * @apiGroup skus
+ * @apiParam {String} identifier of sku, must cast to ObjectId
+ * @apiError NotFound There is no sku with the <code>identifier</code>
+ * @apiErrorExample Error-Response:
+ *     HTTP/1.1 404 Not Found
+ *     {
+ *       "error": "Not Found"
+ *     }
+ */
+var skusIdDeletePack = controller({success: 204}, function (req) {
```

Comment:
Does deleteSkuPackById return any data?  If so, you'll want to return 200, not 204.